lib/containers/container_images.pm

```diff
@@ -123,23 +138,55 @@ sub test_opensuse_based_image {
             my $pretty_version = $version =~ s/-SP/ SP/r;
             my $betaversion    = get_var('BETA') ? '\s\([^)]+\)' : '';
             record_info "Validating", "Validating That $image has $pretty_version on /etc/os-release";
-            validate_script_output("$runtime container run --entrypoint '/bin/bash' --rm $image -c 'grep PRETTY_NAME /etc/os-release' | cut -d= -f2",
-                sub { /"SUSE Linux Enterprise Server ${pretty_version}${betaversion}"/ });
+            if ($runtime =~ /docker/) {
+                # zypper-docker changes the layout of the image
+                validate_script_output("$runtime run --entrypoint /bin/bash $image -c 'grep PRETTY_NAME /etc/os-release' | cut -d= -f2",
```

Comment:
The `--ent	rypoint` issue again…

Reply:
buildah is qute different from docker. https://github.com/containers/buildah/blob/master/docs/buildah-run.md

Reply:
Again, please keep `podman` too.

Reply:
done 946e17650